backend/src/Squidex.Domain.Apps.Entities/Contents/Queries/ContentQueryService.cs

```diff
@@ -210,9 +217,9 @@ private async Task<List<ISchemaEntity>> GetSchemasAsync(Context context)
 
         private static bool HasPermission(Context context, ISchemaEntity schema)
         {
-            var permission = Permissions.ForApp(Permissions.AppContentsRead, context.App.Name, schema.SchemaDef.Name);
+            var permissionReadOwn = Permissions.ForApp(Permissions.AppContentsReadOwn, context.App.Name, schema.SchemaDef.Name);
 
-            return context.Permissions.Allows(permission);
+            return context.Permissions.Allows(permissionReadOwn);
```

Comment:
Can we just add the permission string as parameter and reuse the method in both places (Line 97 and the current usage)

Reply:
Do You mean add and use some private method like this? 

```
private bool SomePrivateMethod(string permission)
{
       return context.Permissions.Allows(permission);
}
```

Reply:
No, we already have this method HasPermission(...) if we add the required permission as a parameter we can use it in both places.